core/jazz-ui/src/app/pages/admin/admin.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'admin',
+  templateUrl: './admin.component.html',
+  styleUrls: ['./admin.component.scss']
+})
+export class AdminComponent implements OnInit {
```

Comment:
Is this file required? I don't seen anything concrete here. Unless the magic lies in lines 3-6